src/quest_manager/tests/test_models.py

```diff
@@ -59,20 +59,46 @@ def test_category_icon(self):
 
     def test_category_url(self):
         self.assertEqual(self.client.get(self.category.get_absolute_url(), follow=True).status_code, 200)
+    
+    def test_current_quests(self):
+        """ Test that the queryset of all quests in a campaign is returned correctly """
+
+        # assert that the current campaign has no quests
+        self.assertEqual(self.category.current_quests().count(), 0)
+
+        # create some quests as part of the test campaign, some are invalid and won't be included
+        baker.make(Quest, campaign=self.category, xp=0)  # included in queryset
+        baker.make(Quest, campaign=self.category, xp=0)  # included in queryset
+        baker.make(Quest, visible_to_students=False, xp=8)  # NOT included in queryset because not visible
+        baker.make(Quest, archived=True, xp=16)  # NOT included in queryset because archived
```

Comment:
based on the code comment, the above two tests should be `campaign=self.category`, otherwise the archived and visible are irrelevant, because they aren't even in the campaign

Reply:
set campaign for the two test quests that needed it.